apps/mobile-wallet/src/components/layout/Screen.tsx

```diff
@@ -29,19 +29,17 @@ const Screen = ({ children, headerOptions, safeAreaPadding, titleAlwaysVisible,
     : {}
 
   return (
-    <KeyboardAvoidingView behavior="height" style={{ flex: 1 }}>
-      <ScreenStyled {...props} style={[props.style, paddingStyle]}>
-        {headerOptions && (
-          <HeaderComponent
-            onBackPress={navigation.canGoBack() ? navigation.goBack : undefined}
-            options={headerOptions}
-            titleAlwaysVisible={titleAlwaysVisible}
-            scrollY={scrollY}
-          />
-        )}
-        {children}
-      </ScreenStyled>
-    </KeyboardAvoidingView>
+    <ScreenStyled {...props} style={[props.style, paddingStyle]}>
+      {headerOptions && (
+        <HeaderComponent
+          onBackPress={navigation.canGoBack() ? navigation.goBack : undefined}
+          options={headerOptions}
+          titleAlwaysVisible={titleAlwaysVisible}
+          scrollY={scrollY}
+        />
+      )}
+      {children}
+    </ScreenStyled>
```

Comment:
This causes #1494 

I tested in both simulators and I can't find any introduced bugs